### ISSUE TITLE
Can not open app on Android S

### ISSUE THREAD

**Issue by Tougee** [signalapp/Signal-Android#11271]
### Bug description
Can not open app on Android S.

### Device info
**Device:** Pixel 3XL
**Android version:** S
**Signal version:** 5.10.6

```
AndroidRuntime E FATAL EXCEPTION: signal-JobManager
E Process: org.thoughtcrime.securesms, PID: 15345
E java.lang.NullPointerException: Attempt to get length of null array
E at com.google.protobuf.GeneratedMessageLite.parseFrom(GeneratedMessageLite.java:1654)
E at org.thoughtcrime.securesms.database.model.databaseprotos.ReactionList.parseFrom(ReactionList.java:649)
E at org.thoughtcrime.securesms.database.helpers.SQLCipherOpenHelper.onUpgrade(SQLCipherOpenHelper.java:1356)
E at net.sqlcipher.database.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:178)
E at net.sqlcipher.database.SQLiteOpenHelper.getReadableDatabase(SQLiteOpenHelper.java:233)
E at net.sqlcipher.database.SQLiteOpenHelper.getReadableDatabase(SQLiteOpenHelper.java:220)
E at org.thoughtcrime.securesms.database.helpers.SQLCipherOpenHelper.getReadableDatabase(SQLCipherOpenHelper.java:1464)
E at org.thoughtcrime.securesms.database.DatabaseFactory.hasTable(DatabaseFactory.java:261)
E at org.thoughtcrime.securesms.database.JobDatabase.dropTableIfPresent(JobDatabase.java:379)
E at org.thoughtcrime.securesms.database.JobDatabase.onOpen(JobDatabase.java:143)
E at net.sqlcipher.database.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:188)
E at net.sqlcipher.database.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:131)
E at org.thoughtcrime.securesms.database.JobDatabase.getReadableDatabase(JobDatabase.java:366)
E at org.thoughtcrime.securesms.database.JobDatabase.getAllJobSpecs(JobDatabase.java:173)
E at org.thoughtcrime.securesms.jobs.FastJobStorage.init(FastJobStorage.java:51)
E at org.thoughtcrime.securesms.jobmanager.JobManager.lambda$new$0(JobManager.java:82)
E at org.thoughtcrime.securesms.jobmanager.JobManager.lambda$new$0$JobManager(Unknown Source:0)
E at org.thoughtcrime.securesms.jobmanager.-$$Lambda$JobManager$Q1aJiMb3MgNpPVzYaxFvnNuEuxg.run(Unknown Source:6)
E at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E at java.lang.Thread.run(Thread.java:920)
```

Another problem: when I use a Chinese phone number, the login sms are not being received.

**Comment by kpuljek** [comment]
Just got an update to 5.10.6 and it crashes on opening as well.

**Comment by tinybike** [comment]
Pixel 2XL here.  Same issue -- Signal was working until today, but now it just crashes immediately on opening.

**Comment by hiqua** [comment]
> Edit: In my case it's related to conversations where the last message was an emoji, or when opening conversations where there's reactions present.

Then https://github.com/signalapp/Signal-Android/commit/1f3e1316900a1473ad79ad219babda0a5025725b might be a fix.

**Comment by greyson-signal** [comment]
Yeah, this is fixed in 5.10.7
